wavefront/server/modules/api_services_module/api_services_module/config/parser.py

```
@@ -106,6 +108,44 @@ def _parse_auth_config(auth_data: Dict[str, Any]) -> AuthConfig:
+        """Parse payload schema configuration."""
+        if not schema_data or 'fields' not in schema_data:
+            return None
+
+        fields = []
+        for field_data in schema_data['fields']:
+            # Validate required fields
+            field_name = field_data.get('name')
+            field_type = field_data.get('type')
+
+            if not field_name:
+                raise ValueError('Payload field missing required attribute: name')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)